orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationPrompt.java

```diff
@@ -32,7 +32,7 @@ public class OrchestrationPrompt {
    * @param message A user message.
    */
   public OrchestrationPrompt(@Nonnull final String message) {
-    messages.add(new ChatMessage().role("user").content(message));
+    messages.add(new UserMessage(message).toChatMessage());
```

Comment:
Instead of changing every usage here, it's probably easier to make the private fields of this object to accept the convenience class, and only convert later once. Then the two apis are also better separated